### PR TITLE
Displays translation options correctly in one line

### LINKED ISSUE #246: "Multi-step translation" option glitches on RTL
image

### PR DESCRIPTION
**Pull request by JPJPJPOPOP**
Fixes #246

### CODE REVIEW

**Comment by JPJPJPOPOP** [comment]
@sushain97 I don't believe that is my fault (i.e. it was already there before). I'll try fixing it though.

File: assets/css/translation.css [hunk 1]
```diff
@@ -87,10 +87,18 @@ html[dir='rtl'] #srcLanguages {
...
 
+#translateOptionsContainer {
+    width: auto;
+}
+
 #translateOptions label {
     clear: both;
 }
 
+html[dir='rtl'] #markUnknown, html[dir='rtl'] #instantTranslation, html[dir='rtl'] #chainedTranslation {
```
Comment (sushain97) [inline assets/css/translation.css]: Can we do `html[dir='rtl'] #translateOptions input[type="checkbox"]` here?

File: index.html.in [hunk 2]
```diff
@@ -225,7 +225,7 @@
                                                 <span data-text="Translate_Webpage">Translate a webpage</span>
                                             </button>
                                         </div>
-                                        <div class="col-lg-3 col-md-6  pull-right">
+                                        <div class="col-lg-3 col-md-6  pull-right" id="translateOptionsContainer">
```
Comment (sushain97) [inline index.html.in]: Mind fixing the extra space here? Also, I think we could just push `#translateOptions` up? I checked the JS and it seems safe.

**Comment by sushain97** [comment]
Hmm, I think a little tweak is necessary?

image

**Comment by JPJPJPOPOP** [comment]
@sushain97 So apparently, punctuation or symbols at the end of a sentence will jump to the front when using rtl. The only reason this is a problem is because the current rtl languages don't have a translation. So either we can wait for someone to translate this for the rtl language or we can add a `&lrm;` at the end of all the translations for `Multi_Step_Translation` so the ending parentheses won't jump to the front.

**Comment by sushain97** [comment]
Interesting... here's a different solution that I think works and is a lot cleaner: Add `dir="..."` attribute whenever substituting a string. We don't actually have any legitimate bidirectional strings as far as I know.

**Comment by JPJPJPOPOP** [comment]
So are you saying that as we localize the strings, set the direction of the text along with it?

**Comment by sushain97** [comment]
Indeed.

**Comment by JPJPJPOPOP** [comment]
@sushain97 That didn't really work. It added to the <span> `direction: ltr`, but it needed to be adding it to the <label>.

**Comment by sushain97** [comment]
Seems to work for me?

image

**Comment by JPJPJPOPOP** [comment]
Ah, woops, was adding it as css

File: assets/js/localization.js [hunk 3]
```diff
@@ -354,6 +354,7 @@ function localizeStrings(stringsFresh /*: boolean */) {
                         }
                         else {
                             elem.html(text);
+                            elem.attr('dir', rtlLanguages.indexOf(locale) === -1 ? 'ltr' : 'rtl');
```
Comment (sushain97) [inline assets/js/localization.js]: You think https://github.com/JPJPJPOPOP/apertium-html-tools/blob/3ac77a33ff60c0623eeeb29b22932345812bf1ee/assets/js/localization.js#L391 could be pulled out and used? I think there are a couple other places that could be easily replaced as well.

Comment (sushain97) [reply assets/js/localization.js]: (the code does seem to work though!)

**Pull request merged**